x/filetree/keeper/access.go

```diff
@@ -37,14 +37,16 @@ func HasEditAccess(file types.Files, user string) bool {
 	json.Unmarshal([]byte(peacc), &jvacc)
 
 	h := sha256.New()
-	h.Write([]byte(fmt.Sprintf("e%d%s", trackingNumber, user)))
+	h.Write([]byte(fmt.Sprintf("e%s%s", trackingNumber, user)))
 	hash := h.Sum(nil)
 
 	addressString := fmt.Sprintf("%x", hash)
+	fmt.Println("@@@@address string is", addressString)
```

Comment:
I'd love if as a standardization we moved away from fmt and used the context's logger instead but will still approve with that

Reply:
My bad--I always remove those before making a PR but I missed this one. Learning how to use context's logger is definitely on the agenda. Many thanks.